tests/acceptance/common.py

```diff
@@ -466,7 +466,8 @@ def prepared_test_build_base(request, bitbake_variables):
     build_dir = tempfile.mkdtemp(prefix="test-build-", dir=os.environ['BUILDDIR'])
 
     def cleanup_test_build():
-        run_verbose("rm -rf %s" % build_dir)
+        if not pytest.config.getoption('--keep-build-dir'):
+            run_verbose("rm -rf %s" % build_dir)
```

Comment:
Definitely +1!

I was always just too much in a hurry to add it myself, but very useful.